### PR TITLE
Payment widget registry

### PR DESCRIPTION
**Pull request by JobDoesburg**
### Summary
After #1673 the admin payment widget was broken

### How to test
Steps to test the changes you made:
1. Check out the payment widget
2. It does show the correct payment amount

### CODE REVIEW

**Review by se-bastiaan — APPROVED** [review]

**Comment by JobDoesburg** [comment]
Wait, I think I made a mistake... I think I should check for `isinstance(self.obj, Payable)` rather than `Payment` right?

**Comment by se-bastiaan** [comment]
> 
> 
> Wait, I think I made a mistake... I think I should check for `isinstance(self.obj, Payable)` rather than `Payment` right?

Thought that was on purpose, because none of the models is a Payable anymore.

**Comment by se-bastiaan** [comment]
Think you may not even need the isinstance

**Comment by JobDoesburg** [comment]
> > Wait, I think I made a mistake... I think I should check for `isinstance(self.obj, Payable)` rather than `Payment` right?
> 
> Thought that was on purpose, because none of the models is a Payable anymore.

I copied it from elsewhere and I was already a bit in doubt...

File: website/payments/widgets.py [hunk 1]
```diff
@@ -17,10 +18,15 @@ def __init__(self, attrs=None, obj=None):
     def get_context(self, name, value, attrs) -> dict:
         context = super().get_context(name, value, attrs)
         if self.obj and not value:
-            context["obj"] = self.obj
+            payable = (
+                self.obj
+                if isinstance(self.obj, Payment)
+                else payables.get_payable(self.obj)
+            )
+            context["obj"] = payable
```
Comment (se-bastiaan) [inline website/payments/widgets.py:26]:
```suggestion
            payable = payables.get_payable(self.obj)
            context["obj"] = payable
```

**Comment by JobDoesburg** [comment]
@se-bastiaan to do this properly, the tests need a fundamental rewrite and I don't understand the whole registry good enough. Can I seduce you to take a look at this for me :)

**Review by se-bastiaan — APPROVED** [review]

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**